package.json

```diff
@@ -15,13 +15,15 @@
   },
   "scripts": {
     "build": "ember build",
+    "install": "cd node_modules/comlink && npm install && npm run build",
     "lint:js": "eslint .",
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each"
   },
   "dependencies": {
     "body-parser": "^1.18.3",
+    "comlink": "CvX/comlink#wrap-chain",
```

Comment:
I know you added some stuff here to get the chaning working. Should we wait for them to merge it or depend on this branch?

Reply:
Continuing from the previous comment: I wouldn’t use this setup. I’d either wait for a comlink release or, in case that gets stuck, I’d push a fork of comlink with chain method calls to npm.